docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,137 @@
+categorie;element;leveren door RvIG;Beperkt zoekresultaat;Persoon GBA variant;;;;;opmerking/issue;
```

Comment:
nu de gba en proxy variant uit elkaar zijn getrokken in aparte bestanden, is waarde "G" bij Beperkt zoekresultaat denk ik niet zo duidelijk meer. Kan denk ik nu makkelijker

Kolomkop "GbaPersoonBeperkt" en in de kolom als waarde "Ja" i.p.v. "G"

Reply:
Is aangepast!